### PR TITLE
David/982 link to portfolio

### PR DESCRIPTION
**Pull request by haveanicedavid**
## Description

Closes: regen-network/regen-registry#982

More changes here than I would have liked, but:
- creates an `AccountLink` component which will route to `/ecocredits/accounts/:accountId` for regen addresses (in the same tab), and the appropriate block explore address (in a new tab) for others.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood` and `hambach` (see below)

#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by mhagel — APPROVED** [review]

File: web-registry/src/components/atoms/AccountLink.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+}: AccountLinkProps): JSX.Element => {
+  const isRegen = address.startsWith('regen');
+  const content = !!children ? children : truncate(address);
+  if (!isRegen) {
+    return (
+      <MuiLink
+        {...linkProps}
+        href={getAccountUrl(address)}
+        target={linkProps.target || '_blank'}
+        rel="noopener noreferrer"
```
Comment (wgwz) [inline web-registry/src/components/atoms/AccountLink.tsx:33]: we might not need this in 2022, refs:
- https://stackoverflow.com/a/50709724/4028706
but then again maybe lighthouse scores are affected:
- https://web.dev/external-anchors-use-rel-noopener/

**Review by wgwz — APPROVED** [review]
LGTM, just added a nit

File: web-registry/src/components/atoms/AccountLink.tsx (hunk 1 above)
Comment (haveanicedavid) [reply web-registry/src/components/atoms/AccountLink.tsx:33]: Oh I didn't know this! interesting. 

Still seems like I might just add, given it doesn't have full browser support yet. Still nice to know, it's a bit of habit at this point

**Comment by haveanicedavid** [comment]
@mergifyio backport v4

**Pull request merged**